sorc/ocean_merge.fd/merge_lake_ocnmsk.F90

```diff
@@ -18,8 +18,8 @@ program merge_lake_ocnmsk
 
   implicit none
 
-  character(len=120) :: pth1
-  character(len=120) :: pth2,pth3
+  character(len=200) :: pth1
+  character(len=200) :: pth2,pth3
```

Comment:
I see different values of  atleast 1,2,3,4,6,20 ,32100,128160,256,512 in different codes.
Is there a need to standardize or extend them as we need?

Reply:
I don't know if I would standardize length of character strings. Rather, when we review future PRs, we should keep in mind that any string used for directory paths should be 'large'.